### PR TITLE
CREATED THE GAME ELEPHANT ANT MAN

### LINKED ISSUE #4015: [New game]: man,elephant,ant
### 🎮 Game Request

Game logic and basic description: another version of rock paper scissors but with a cuter look. It is inspired by the good old bedtime story of a elephant and an ant where the ant teaches the elephant a lesson.

### Point down the features

Game points: play with the computer , the computer randomly chooses one of the 3 options when the user does. The won who wins will receive a point. It is a never ending game and can be played with however times one wants. in the game:
the man kills the ant while the ant bites the elephant furthermore the man is crushed by the elephant.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by tanishkaa08**
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes #4015 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @tanishkaa08 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by tanishkaa08** [comment]
Please review it.

File: assets/images/screenshot.png
Comment (ishita-43) [inline assets/images/screenshot.png:1]: rename your image name as per naming conventions in contribution guidelines.

File: README.md [hunk 1]
```diff
@@ -320,13 +321,14 @@ This repository also provides one such platforms where contributers come over an
...
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
+|[Elephant_ant_man](https://github.com/tanishkaa08/GameZone/tree/main/Games/Elephant_ant_man)    |
+
 
 
 
  </center>
 
 <br>
-<br>
```
Comment (ishita-43) [inline README.md:329]: add this line again

File: README.md [hunk 2]
```diff
@@ -320,13 +321,14 @@ This repository also provides one such platforms where contributers come over an
 |[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game)   |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
+|[Elephant_ant_man](https://github.com/tanishkaa08/GameZone/tree/main/Games/Elephant_ant_man)    |
```
Comment (ishita-43) [inline README.md:324]: follow other entries format while providing the link it should start with kunjgit instead of tanishkaa

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Hoping you will make desired changes soon!!!

**Review by ishita-43 — APPROVED** [review]
Conflicts resolved

**Comment by kunjgit** [comment]
Thank you  @tanishkaa08 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**